official/resnet/keras/keras_cifar_benchmark.py

```diff
@@ -153,6 +153,12 @@ def benchmark_graph_1_gpu_no_dist_strat(self):
     FLAGS.num_gpus = 1
     FLAGS.enable_eager = False
     FLAGS.turn_off_distribution_strategy = True
+    # Test errors only if all 1_gpu test are run together. Adding
+    # FLAGS.model_dir to this test fixes the problem for unknown reason.
+    # Error is: tensorflow.python.framework.errors_impl.NotFoundError:
+    # Resource localhost/logdir:/tmp/cifar10_model/
+    # N10tensorflow22SummaryWriterInterfaceE does not exist.
+    FLAGS.model_dir = self._get_model_dir('benchmark_graph_1_gpu_no_dist_strat')
```

Comment:
It works just fine if I run just this test, but if fails if I run all the tests together.  It does  not make a lot of sense.  Until it is resolved none of the test run.  I am fine setting a folder for all of them, I suspect it might be an issue with all of them having the same path.

Reply:
I am not setting them because shinning did not set a model_dir and these tests are copied from his code.  I would have set it so if there was an event log or something I would save it.  Of course that covers up this weird error.

Reply:
Let's go with setting it in all of them.

Reply:
All fixed.  Sorry for the delay.